src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -93,37 +93,29 @@ private function findTransitions(Definition $definition)
         return $transitions;
     }
 
-    private function addPlaces(array $places)
+    protected function addPlaces(array $places)
     {
         $code = '';
 
         foreach ($places as $id => $place) {
-            $code .= sprintf("  place_%s [label=\"%s\", shape=circle%s];\n",
-                $this->dotize($id),
-                $id,
-                $this->addAttributes($place['attributes'])
-            );
+            $code .= sprintf("  place_%s [label=\"%s\", shape=circle%s];\n", $this->dotize($id), $id, $this->addAttributes($place['attributes']));
         }
 
         return $code;
     }
 
-    private function addTransitions(array $transitions)
+    protected function addTransitions(array $transitions)
     {
         $code = '';
 
         foreach ($transitions as $place) {
-            $code .= sprintf("  transition_%s [label=\"%s\", shape=box%s];\n",
-                $this->dotize($place['name']),
-                $place['name'],
-                $this->addAttributes($place['attributes'])
-            );
+            $code .= sprintf("  transition_%s [label=\"%s\", shape=box%s];\n", $this->dotize($place['name']), $place['name'], $this->addAttributes($place['attributes']));
         }
 
         return $code;
     }
 
-    private function findEdges(Definition $definition)
+    protected function findEdges(Definition $definition)
```

Comment:
IMO, this should not be protected extension points. This seems like the wrong way to reuse part of the GraphvizDumper (fixing the dumping of transitions to stop assuming unique names might require refactoring private methods, but making them protected forbids this.